static/styles/styles.css

```
@@ -131,6 +145,57 @@ font-size: 24px;
 
     /* Is there a better way to do this? */
     .headerImage {
-        width: 110%;
+        width: 325px !important; /* for mobile, it's better to fix the width */
```

Not a big fan of `!important`, but we can let it slide for now.

This should work:

```suggestion
        max-width: 325px
```